LocationSync/LocationSync.cpp

```diff
@@ -27,6 +27,25 @@ namespace Plugin {
     static Core::ProxyPoolType<Web::Response> responseFactory(4);
     static Core::ProxyPoolType<Web::JSONBodyType<LocationSync::Data>> jsonResponseFactory(4);
 
+    namespace {
+        const string TZ_FILE = "/opt/persistent/timeZoneDST";
+
+        string readTextFile(const string &filename) {
```

Comment:
As you might have experienced, the Metro style is using PascalCase method naming, so this should be called ReadTextFile